pandas/core/internals/blocks.py

```diff
@@ -231,11 +231,12 @@ def get_values(self, dtype=None):
             return self.values.astype(object)
         return self.values
 
-    def get_block_values(self, dtype=None):
+    def get_block_values_for_json(self) -> np.ndarray:
         """
-        This is used in the JSON C code
+        This is used in the JSON C code.
         """
-        return self.get_values(dtype=dtype)
```

Comment:
Why not keep it as `get_values()` (but remove the dtype, since that is not used) ? The get_values already handles the reshape, avoiding the need to keep track of it here as well.

Reply:
Because `get_values` is next up on the chopping block